rebar.config

```diff
@@ -1,7 +1,17 @@
 {erl_opts, [debug_info]}.
 {deps, [{escalus, ".*", {git, "git://github.com/esl/escalus.git", {branch, "master"}}}]}.
 
-{relx, [{release, {mercutio, "0.0.1"},[mercutio, runtime_tools]},
+{relx, [{release, {mercutio, "0.0.1"},[
+           mercutio,
+           p1_utils,
```

Comment:
Could you drop a note in the commit message why this is needed?

Reply:
8 months ago Nelson added `fast_tls` dependency to `escalus`, `fast_tls` in its turn has `p1_utils` dependency but doesn't list it in [*.app.src](https://github.com/processone/fast_tls/blob/master/src/fast_tls.app.src#L29) file so it's not automatically resolved, and we have to add it to relx configuration.

I've created an [issue](https://github.com/processone/fast_tls/issues/50) for fast_tls